pkg/pipeline/CdHandler.go

```diff
@@ -731,27 +738,34 @@ func (impl *CdHandlerImpl) GetRunningWorkflowLogs(environmentId int, pipelineId
 		impl.Logger.Errorw("error while fetching cd pipeline", "err", err)
 		return nil, nil, err
 	}
-
-	serverUrl := env.Cluster.ServerUrl
 	configMap := env.Cluster.Config
-	bearerToken := configMap["bearer_token"]
+	clusterConfig := util.ClusterConfig{
+		Host:                  env.Cluster.ServerUrl,
+		BearerToken:           configMap["bearer_token"],
```

Comment:
same here ,create constants. check these everywhere in your code and create constants